react-components/tests/tests-utilities/architecture/commands/TestInputCommand.ts

```diff
@@ -0,0 +1,65 @@
+import { type TranslationInput } from '../../../../src/architecture';
```

Comment:
Same comment as here:

https://github.com/cognitedata/reveal/pull/5120#discussion_r2095420087

(This being the second time this pops up today I do think you should look into it)

Reply:
It seems to me that the `vite.config.ts` in the project only looks for things referenced by the `src/index.ts`. This file is also put in the `tests-utilities` folder, which is only intended for test-specific code. But are you thinking we should have some way of explicitly excluding this in the build output?

Reply:
tsconfig.*.js(on)

Reply:
It was suggested that we instead do this in the `vite` config. The `exclude`-field in the `tsconfig` doesn't guarantee exclusion like we want. In any case, we will do this in a separate PR